diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeTabs/index.js
index a0e09d4..8ab8ea3 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeTabs/index.js
@@ -240,9 +240,12 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   const { className } = props;
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     {
       className: (0, clsx_1.default)(
@@ -252,6 +255,7 @@ function TabsComponent(props) {
     },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function CodeTabs(props) {
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiTabs/index.js
index 660f490..508823d 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiTabs/index.js
@@ -250,12 +250,16 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     { className: "openapi-tabs__container" },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function ApiTabs(props) {
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/DiscriminatorTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/DiscriminatorTabs/index.js
index f536508..3a5dfa7 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/DiscriminatorTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/DiscriminatorTabs/index.js
@@ -228,12 +228,16 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     { className: "openapi-tabs__container" },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function DiscriminatorTabs(props) {
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/MimeTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/MimeTabs/index.js
index 93c0559..6b420ee 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/MimeTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/MimeTabs/index.js
@@ -261,12 +261,16 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     { className: "tabs-container" },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function MimeTabs(props) {
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/OperationTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/OperationTabs/index.js
index 6c7c795..8162267 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/OperationTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/OperationTabs/index.js
@@ -223,12 +223,16 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     { className: "tabs-container" },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function OperationTabs(props) {
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/SchemaTabs/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/SchemaTabs/index.js
index d1924da..1752351 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/SchemaTabs/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/SchemaTabs/index.js
@@ -242,12 +242,16 @@ function TabContent({ lazy, children, selectedValue }) {
   );
 }
 function TabsComponent(props) {
-  const tabs = (0, internal_1.useTabs)(props);
+  const tabs = (0, internal_1.useTabsContextValue)(props);
   return react_1.default.createElement(
+    internal_1.TabsProvider,
+    { value: tabs },
+    react_1.default.createElement(
     "div",
     { className: "openapi-tabs__schema-container" },
     react_1.default.createElement(TabList, { ...props, ...tabs }),
     react_1.default.createElement(TabContent, { ...props, ...tabs })
+  )
   );
 }
 function SchemaTabs(props) {
