diff --git a/node_modules/docusaurus-plugin-openapi-docs/lib/openapi/openapi.js b/node_modules/docusaurus-plugin-openapi-docs/lib/openapi/openapi.js
index 09c573d..f4ac61e 100644
--- a/node_modules/docusaurus-plugin-openapi-docs/lib/openapi/openapi.js
+++ b/node_modules/docusaurus-plugin-openapi-docs/lib/openapi/openapi.js
@@ -130,6 +130,7 @@ function createItems(openapiData, options, sidebarOptions) {
             frontMatter: {
                 description: splitDescription
                     ? splitDescription[0]
+                        .replace(/\\/g, "\\\\")
                         .replace(/((?:^|[^\\])(?:\\{2})*)"/g, "$1'")
                         .replace(/\s+$/, "")
                     : "",
@@ -228,6 +229,7 @@ function createItems(openapiData, options, sidebarOptions) {
                 frontMatter: {
                     description: splitDescription
                         ? splitDescription[0]
+                            .replace(/\\/g, "\\\\")
                             .replace(/((?:^|[^\\])(?:\\{2})*)"/g, "$1'")
                             .replace(/\s+$/, "")
                         : "",
@@ -350,6 +352,7 @@ function createItems(openapiData, options, sidebarOptions) {
                 frontMatter: {
                     description: splitDescription
                         ? splitDescription[0]
+                            .replace(/\\/g, "\\\\")
                             .replace(/((?:^|[^\\])(?:\\{2})*)"/g, "$1'")
                             .replace(/\s+$/, "")
                         : "",
@@ -411,6 +414,7 @@ function createItems(openapiData, options, sidebarOptions) {
                     frontMatter: {
                         description: splitDescription
                             ? splitDescription[0]
+                                .replace(/\\/g, "\\\\")
                                 .replace(/((?:^|[^\\])(?:\\{2})*)"/g, "$1'")
                                 .replace(/\s+$/, "")
                             : "",
@@ -450,6 +454,7 @@ function createItems(openapiData, options, sidebarOptions) {
                 frontMatter: {
                     description: splitDescription
                         ? splitDescription[0]
+                            .replace(/\\/g, "\\\\")
                             .replace(/((?:^|[^\\])(?:\\{2})*)"/g, "$1'")
                             .replace(/\s+$/, "")
                         : "",
